### PR TITLE
Remove can-i-get-a-british-passport Smart Answer

### PR DESCRIPTION
**Pull request by chrisroos**
* Added on 7th Mar 2012 - ae39fa485477592718df060b49f054c691788c41
* Last updated on 7th Mar 2012 - 1f27b1d19dad7e86ed161bcce49c7fb321930cfe
* Never published
* No associated test

Gwen and Alex have confirmed that this can be removed.

### CODE REVIEW

**Comment by chrisroos** [comment]
My motivation for opening this pull request is to run the tests and to see whether people spot anything else that can be removed in this branch.

**Comment by floehopper** [comment]
LGTM. I can't see anything else which can be removed.

File: lib/smart_answer_flows/can-i-get-a-british-passport.rb [hunk 1]
```diff
@@ -1,75 +0,0 @@
-status :draft
-satisfies_need "100983"
```
Comment (tadast) [inline lib/smart_answer_flows/can-i-get-a-british-passport.rb]: This makes me think: do they need too update anything in Maslow (user needs tracker)? Mind checking with Gwen?

**Comment by tadast** [comment]
:shipit:

File: lib/smart_answer_flows/can-i-get-a-british-passport.rb (hunk 1 above)
Comment (chrisroos) [inline lib/smart_answer_flows/can-i-get-a-british-passport.rb]: Good question. I'll find out.

Comment (chrisroos) [inline lib/smart_answer_flows/can-i-get-a-british-passport.rb]: @tadast: Liz says, "I think we're fine - at least 3 of those would have content besides the smart answer, so if there are any needs in maslow, they'll be covered."

**Pull request merged**